makeservices/makemysql-real

```diff
@@ -1,153 +1,78 @@
-#!/usr/bin/python
-#
-# This script creates a MySQL database with the same name as the username
-# of the user who runs this program. To prevent the user from taking control
-# of this program and e.g., causing the program to dump core and reveal the
-# MySQL root password, this program should be run under another account,
-# preferably that of the MySQL user.
-#
-# The password set for this new account is randomly generated; it is not
-# user-selectable. This is to prevent compromised MySQL-backed web apps from
-# also compromising the shell account password. Note that the MySQL server
-# should also be patched to prevent the user from later changing the password.
-#
-# The MySQL root password and the hostname of the MySQL server are read from
-# a config file; the path to it is stored in the CONF_FILE global. This file
-# must be readable only by the setuid user! The Python ConfigParser module
-# is used to parse the config file, so the format is basically the Windows
-# INI format.
-#
-from __future__ import print_function
-
+#!/usr/bin/python3
+""" Creates a MySQL database with the same name as the username of the user who
```

Comment:
nit: `#!/usr/bin/env python` for shebang
nit: no space at the start of docstrings.